js/infobar.js

```diff
@@ -1,4 +1,4 @@
-$(function() {
+(function(window) {
```

Comment:
I wouldn't recommend passing in `window` in this fashion. Either rename it to something shorter (e.g. `w`) in the IIFE, or don't pass it in at all (as `window` should always refer to the global).